components/Mediator.vue

```diff
@@ -285,6 +297,42 @@ export default {
       deferredCredentialOperation.resolve(null);
       await navigator.credentialMediator.hide();
     },
+    async webShare() {
+      const {credential} = this;
+      let file;
+      if(credential.type === 'web' &&
+         credential.dataType === 'VerifiablePresentation') {
+        const vp = credential.data;
+        const blob = new Blob(
+          [JSON.stringify(vp, null, 2)],
+          {type: 'text/plain'}
```

Comment:
Aha, good question. So, it turns out that you can't share /files/ with a mobile browser. (Which, I realize, is weird, but they can only handle sharing links & images etc). But it just gives "Chrome cannot handle this link" error.
On the other hand, you CAN select things like Google Drive. In which case, TXT becomes a big advantage (drive doesn't allow html files, but allows txt files just great).

Reply:
Hmm, maybe this is ok because if we could send a the MIME type we really wanted (either JSON or something even more specific) we wouldn't have that user warning feature anyway.